src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -141,6 +141,42 @@ export const mapTokenUi = ({
 	})
 });
 
+/** Function to sum the balances of two tokens.
+ *
+ * If the decimals of the tokens are the same, the balances are added together.
+ * If the decimals are different, the function returns null.
```

Comment:
Maybe I'm misleading it but,`null` is maybe not specified as return type of the function?

Reply:
I defined the return type as `TokenUi['balance']` which definition is 

![Screenshot 2024-10-23 at 13 24 26](https://github.com/user-attachments/assets/159de022-3e1f-49cb-8770-886074da3b76)

Reply:
I have a few tests to check the different returns for `null` and `undefined`

Reply:
Cool